spec/helpers/measures_helper_spec.rb

```diff
@@ -68,4 +68,48 @@
       end
     end
   end
+
+  describe '#reformat_duty_expression' do
+    subject { reformat_duty_expression expression }
+
+    context 'with simple percentage' do
+      let(:expression) { '<span>20.00</span> % ' }
+
+      it { is_expected.to have_css 'span.duty-expression > span', text: '20.00 %' }
+    end
+
+    context 'with abbreviation' do
+      let :expression do
```

Comment:
We do a mixture of DSL let and non-DSL lets without parentheses, here. Not blocking but I reckon we should just pick one

Reply:
Not sure I follow - pattern I'm using is brackets for methods with inline blocks, and without brackets if using multiline `do`/`end` - same as for the `context` blocks?

Reply:
I've not seen foregoing parentheses on let regardless of multi/single line style in the wild/other projects. Better specs do not mention that style, etiher. Is fine just jarring if you're not used to it. I always just use parentheses so I guess we have two different styles there